src/components/ActionControls/DrawingControls.scss

```diff
@@ -2,28 +2,19 @@
 
 .ba {
     .ba-action-controls-draw {
-        display: flex;
-    }
+        $button-spacing: 5px;
 
-    .ba-action-controls-draw .btn-plain {
-        flex-grow: 1;
-        padding-left: 5px;
-        padding-right: 5px;
+        display: flex;
+        margin-left: -#{$button-spacing};
+        margin-right: -#{$button-spacing};
 
-        .fill-color {
-            fill: $sevens;
+        .btn-plain {
+            flex-grow: 1;
+            padding-left: $button-spacing;
+            padding-right: $button-spacing;
         }
-    }
-
-    .icon-trash {
```

Comment:
This not needed anymore?

Reply:
This can be removed since we're using the trash icon from BRUI now :)

Reply:
The margin was being used to simulate vertical alignment. The new styles make it unnecessary.